### ISSUE TITLE
Rendering related exception on opening a solution

### ISSUE THREAD

**Issue by tomvoros** [kraftboy/prochint-pub#1]
![image](https://user-images.githubusercontent.com/804053/60729511-e0b8a080-9f10-11e9-8cde-69436e69f75e.png)

I'm getting the error popup above every time I open a solution in VS. The actual exception from the log is:

```
System.ArgumentException: Width and Height must be non-negative.
   at System.Windows.Rect..ctor(Double x, Double y, Double width, Double height)
   at ProcHint.ProcHintAdornment.UpdateImage(Double width, Double height) in D:\a\1\s\ProcHint\ProcHintAdornment.cs:line 196
   at ProcHint.ProcHintAdornment.UpdateImageAndText() in D:\a\1\s\ProcHint\ProcHintAdornment.cs:line 130
   at ProcHint.ProcHintAdornment.Redraw() in D:\a\1\s\ProcHint\ProcHintAdornment.cs:line 230
   at ProcHint.ProcHintAdornment.OnSizeChanged(Object sender, EventArgs e) in D:\a\1\s\ProcHint\ProcHintAdornment.cs:line 94
   at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.RaiseEvent[TArgs](Object sender, EventHandler`1 eventHandlers, TArgs args)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

I think this started happening after a recent VS update. I'm currently on version 15.9.12.

**Comment by tomvoros** [comment]
Additional info: I have the ProcHint panel pinned in my debug layout (not pinned in the default layout), and it still works fine despite the exception on startup.

**Comment by kraftboy** [comment]
fixed in 1.6.2
